microsoft/iot-cardboard-js
# OAT: Adding context initialization resiliency

### Summary of changes 🔍 
Refactoring how we bootstrap the OAT context. It will now be more resilient to missing local storage values and will automatically create a project if none exists.

Logic:
- Retrieve last used project id from local storage
- Check if that id exists in the stored files
- If the file is found in storage, use it
- If the file is NOT found in storage, try to use the first file in storage
- If no other files are in storage, create a new project

### Testing 🧪
- use the main oatPage story
- Delete all your oat keys and load the app --> should get "new ontology" project
- Delete only your `oat-files` key but keep the `oat-last-project` key --> should get "new ontology" project
- Delete only your `oat-last-project` key --> should load the first project in storage

### Checklist ✔️
